nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -193,6 +214,8 @@ def _execute_op(op_address: 'OperationAddress',
             is_called_inside_nncf_module = isinstance(ctx.get_current_module(), _NNCFModuleMixin)
             node = ctx.maybe_add_node(processed_input, tensor_metas, op_address,
                                       layer_attrs, ignored_algos, is_called_inside_nncf_module)
+            if node is not None:
+                _add_operation_attributes(node, ctx)
```

Comment:
Have to override existing attributes for operations that require graph traversal in order to find layer attributes.
input and output features for `matmul` is extracted from the shape of second input.